sympy/interactive/session.py

```diff
@@ -416,11 +416,12 @@ def init_session(ipython=None, pretty_print=True, order=None,
                 raise RuntimeError("IPython is not available on this system")
             ip = None
         else:
-            if V(IPython.__version__) >= '0.11':
-                try:
-                    ip = get_ipython()
-                except NameError:
-                    ip = None
+            version = V(IPython.__version__)
+            if '1.0' > version >= '0.11':
+                ip = IPython.core.ipapi.get()
```

Comment:
The docstring of ipapi says:
> This module is *completely* deprecated and should no longer be used for
any purpose.

Instead of this, it would probably suffice to add `from IPython import get_ipython` after `try:` above and then change `NameError` to `ImportError`.